modules/admin-home/rest/whats-new.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace HelloTheme\Modules\AdminHome\Rest;
+
+use Elementor\WPNotificationsPackage\V110\Notifications as ThemeNotifications;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Whats_New extends Rest_Base {
+
+	public function get_theme_notifications(): ThemeNotifications {
+		static $notifications = null;
+
+		if ( null === $notifications ) {
+			require get_template_directory() . '/vendor/autoload.php';
+
+			$notifications = new ThemeNotifications(
```

Comment:
we need the notification SDK instantiated not only on Rest, please see what we have on Hello Biz

Reply:
@nuritsha done